release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -579,7 +585,11 @@ In the table below, features are marked with the following statuses:
 [id="ocp-4-5-known-issues"]
 == Known issues
 
-
+* The deployment Pod cannot mirror images using a digest ID if the imagestream
```

Comment:
I think this bug text is a little confusing, and the standard format doesn't work very well for deferred bugs.

I'd be tempted to say something like:

> Because the `ImageContentSourcePolicy` for image registry pull-through is not yet supported, the deployment Pod cannot mirror images by using a digest ID if the imagestream has the pull-through policy enabled. In this case, an `ImagePullBackOff` error displays.

But if that's not feasible, here are some specific suggestions.

s/using/by using

> ! using n
To avoid ambiguity, replace this gerund with either “by using” or “that uses.

Reply:
I think the most important thing to focus on is clarity for the known issues, using the standard format only as a guide. With that said, I'll go with your suggestion, since I think it's less choppy. :+1: